docs-source/content/faq/external-clients.md

```diff
@@ -0,0 +1,213 @@
+---
+title: "External WebLogic Clients"
+date: 2019-11-21T21:23:03Z
+draft: false
+weight: 6
+---
+
+## WIP - This is a DRAFT!
+
+> __This document is an early draft.__
+> __Do not merge.__
+> __There is no need to review yet.__
+
+
+### Approaches
+
+There are at least three approaches for external WebLogic EJB or JMS client access to a Kubernetes hosted WebLogic cluster. These include [Federation](#federation), [Load Balancer Tunneling](#load-balancer-tunneling), and [Kubernetes Node Ports](#kubernetes-node-ports).
+
+> __NOTE:__ This FAQ is for remote EJB and JMS clients - not JTA clients. The Operator does not currently support external WebLogic JTA access to a WebLogic cluster, as (a) external JTA access requires each server in the cluster to be individually addressable by the client, but this conflicts with (b) the current Operator requirement that a network channel in a cluster have the same port across all servers in the cluster.
+
+#### Federation
+
+- Deploy the remote clients inside another Kubernetes cluster.
+
+- Federate the two clusters so that they effectively share the same DNS namespace. See [Federation](https://kubernetes.io/docs/concepts/cluster-administration/federation/) in the Kubernetes documentation.  Note that federation is a relatively new Kubernetes feature as of 2019.
+
+- If the addresses (URLs) for remote access from the client cluster to the original cluster are not the same as the addresses used by clients running within the original cluster itself, then it is often necessary to configure a custom channel for the T3 protocol in the original cluster.  See [Adding a WebLogic Custom Channel](#adding-a-weblogic-custom-channel).
+
+
+#### Load Balancer Tunneling
+
```

Comment:
There should be a sentence before the list to explain what the list is

Reply:
Seems like you assume that people know what these are, and just jump right into setup instructions

Reply:
done